static_src/components/user_list.jsx

```diff
@@ -134,8 +134,6 @@ export default class UserList extends React.Component {
 
     if (this.props.empty) {
       content = this.emptyState;
-    } else if (this.props.users.length === 1) {
```

Comment:
I didn't think the `onlyOneState` makes sense anymore. If the user is an orga manager, we will want to show the invite component, and if they are a space manager, we want to show the message telling them to ask their org manager to invite users. 

Maybe we need to show it regardless.